docker/build-tools/Dockerfile

```diff
@@ -628,10 +628,13 @@ ENV CARGO_HOME=/home/.cargo
 ENV RUSTUP_HOME=/home/.rustup
 # hadolint ignore=DL4006
 RUN curl --proto '=https' -v --tlsv1.2 -sSf https://sh.rustup.rs | \
-    sh -s -- -y -v --default-toolchain ${RUST_VERSION} --profile minimal --component rustfmt clippy &&\
-    /home/.cargo/bin/rustup default ${RUST_VERSION} &&\
+    sh -s -- -y -v --default-toolchain ${RUST_VERSION} --profile minimal && \
+    /home/.cargo/bin/rustup default ${RUST_VERSION} && \
+    /home/.cargo/bin/rustup component add rustfmt clippy llvm-tools && \
```

Comment:
Per explanation in https://github.com/istio/tools/pull/3091 I think adding `llvm-tools` here is fine to include.

Reply:
We have `llvm-tools`, I am just using the full path as the way it is defined here still causing the same test failures - https://github.com/istio/tools/pull/3091/files